### PR TITLE
[WIP] Feature/election lookup

### PR DESCRIPTION
**Pull request by jmcarp**
Prototype of #354.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -117,3 +118,9 @@ gulp.task('build-js', function() {
   .bundle()
   .pipe(callback('static/js/common.js'));
 });
+
+gulp.task('test', function() {
+  return mochify('./tests/unit/**.js')
```
Comment (msecret) [inline gulpfile.js]: I was thinking we'd keep dir organization here mirrored with the actual js. SO in this case, we'd have `tests/unit/modules` or `tests/unit/pages`. What do you think of that?

File: tests/unit/election-lookup.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+var sinonChai = require('sinon-chai');
+var expect = chai.expect;
+chai.use(sinonChai);
+
+var $ = require('jquery');
+var _ = require('underscore');
+
+var ElectionLookup = require('../../static/js/pages/election-lookup').ElectionLookup;
+
+_.extend(window, {
```
Comment (msecret) [inline tests/unit/election-lookup.js]: This should eventually be moved into a location and then all tests will import it.

File: gulpfile.js (hunk 1 above)
Comment (jmcarp) [inline gulpfile.js]: Agreed. I separated the election lookup logic into a resuable module (modules/election-lookup) and a page that imports and instantiates the module (pages/election-lookup), so the tests have moved to tests/unit/modules/election-lookup.

File: tests/unit/election-lookup.js (hunk 2 above)
Comment (jmcarp) [inline tests/unit/election-lookup.js]: Agreed--I was thinking of deferring this until we have more tests.

**Comment by jmcarp** [comment]
Updated with style improvements from @noahmanger. Want to merge now for testing tomorrow? If not, I can deploy this branch directly to the dev server.

**Comment by noahmanger** [comment]
Looks great. Let me know when you push it up.

**Pull request merged**